cmd/cel-key/node_types.go

```diff
@@ -0,0 +1,52 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	flag "github.com/spf13/pflag"
+
+	sdkflags "github.com/cosmos/cosmos-sdk/client/flags"
+)
+
+var (
+	nodeDirKey = "node.type"
+
+	bridgeDir = "~/.celestia-bridge/keys"
+	fullDir   = "~/.celestia-full/keys"
+	lightDir  = "~/.celestia-light/keys"
+)
+
+func DirectoryFlags() *flag.FlagSet {
+	flags := &flag.FlagSet{}
+	flags.String(nodeDirKey, "", "Sets key utility to use the node type's directory (e.g. "+
+		"~/.celestia-light if --node.type light is passed.")
+
+	return flags
+}
+
+func ParseDirectoryFlags(cmd *cobra.Command) error {
```

Comment:
Any chance we can make dir customizable as we extracted this into a separate bin?

Reply:
User can do `--keyring-dir`